apps/camera/js/lib/camera/focus.js

```diff
@@ -81,6 +81,7 @@ Focus.prototype.configureFocusModes = function(mode) {
   this.faceDetection =
     faceDetectionUserEnabled && faceDetectionSupported && mode === 'picture';
   this.mozCamera.addEventListener('focus', this.onAutoFocusStateChange);
+  this.updateFocusState('undefined');
```

Comment:
I don't see how this does anything. The if statement at 160 should just cause this to return immediately because "'undefined' === 'focusing' || this.focusState === 'focusing'" will return false...?